module/src/tracks.ts

```diff
@@ -53,6 +58,51 @@ export class Tracks {
     }
   }
 
+  // Return all / filtered vessels and their tracks
+  async getAll(params?: QueryParameters, position?: Position): Promise<VesselCollection> {
+    const res: VesselCollection = {}
+    let keys = Object.keys(this.tracks)
+    for (let k of keys) {
+      await this.get(k)
+        .then((t: LatLngTuple[]) => {
+          // filter results based on supplied params
+          if (this.applyFilters(t, params, position)) {
+            res[k] = t
+          }
+        })
+        .catch((err) => {
+          console.log(err)
+        })
+    }
+    return Promise.resolve(res)
+  }
+
+  // returns true if last track point passes filter tests
+  applyFilters(t: LatLngTuple[], params?: QueryParameters, vesselPosition?: Position): boolean {
+    let result: boolean = true
+    if (params && Object.keys(params).length != 0) {
+      let lastPoint: any = t.length != 0 ? t[t.length - 1] : null
+      // within supplied bounded area
+      if (params.geobounds) {
+        if (lastPoint && inBounds(lastPoint, params.geobounds)) {
+          result = result && true
+        } else {
+          result = false
+        }
+      }
+      // within supplied radius of vessel position
+      if ((this.config.maxRadius || params.radius) && vesselPosition) {
```

Comment:
If set maxRadius now overrides the query param.  Should it be the other way around and rename it to `defaultRadius`? Or am I missing someting here?

Reply:
The logic here is:
```
if `?radius=xx` is provided then use xx
else if plugin config maxRadius has a non-zero value use it
else return everything
```